API/NoDocs/Microsoft.Extensions.Diagnostics.ResourceMonitoring/CountersSetup.cs

```diff
@@ -0,0 +1,10 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.ResourceMonitoring'
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+public static class CountersSetup
+{
+    public static void PreparePerformanceCounters();
```

Comment:
What is this for?

Reply:
This is to install the actual counters on Windows, afaik

Reply:
This is needed in order to use the performance counter publisher exposed by this package.

Reply:
This API has been removed.